install-dev/upgrade/upgrade.php

```diff
@@ -34,6 +34,60 @@
 $engineType = 'ENGINE_TYPE';
 define('PS_IN_UPGRADE', 1);
 
+$incompatibleModules = [
```

Comment:
At first I wanted to be able to fetch the information from the addons api, but that's not easy. That's why I finally choose to put a hardcoded list of modules. But we could fix it properly later, once it'll be easy to fetch the compatibility info from the addons api, based on the module name.